### PR TITLE
Portfolio: Setup and mobile first

### PR DESCRIPTION
**Pull request by vanelnw**
#Microverse-project-1

in this project we : 

- configure linters file
- Add index.html file and create structure of project
- Add index.css and design the interface
- Add a documentation of project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# Vanel-Portfolio
\ No newline at end of file
+# Microverse-project-1
+
+Description the project.
+
```
Comment (GiuseppeTG) [inline README.md]: - [ ] Kindly update your readme so that it contains details about your project (e.g. the description). 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>title</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="toolbar">
+        <span>John Doe</span>
```
Comment (GiuseppeTG) [inline index.html]: - [ ] The logo should have a link (`<a>` tag). 
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <img src="./assets//menu.png" />
+      </div>
+
```
Comment (GiuseppeTG) [inline index.html]: - [ ] Your `<header>` tag should wrap just the toolbar, not both sections. 
Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        </p>
+        <div class="socials">
+          <img src="./assets/github.png" alt="github" />
+          <img src="./assets/linkedin.png" alt="linkedin" />
+          <img src="./assets//Vector.png" alt="vector1" />
+          <img src="./assets//twitter.png" alt="twitter" />
+          <img src="./assets//Vector2.png " alt="vector2" />
+        </div>
+
+        <img class="imagebg" src="./assets//imagebg.png" alt="imagebg" />
```
Comment (GiuseppeTG) [inline index.html]: - [ ]  If an image is not part of the content (aesthetic reasons) please use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info 🤓 

Then you should also be able to position this background image and have a very nice result 😄

File: style.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+.headline {
+  position: relative;
+  background-color: black;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  width: 375px;
+}
```
Comment (GiuseppeTG) [inline style.css]: - [ ] As you can see in this image your section has a white margin around it:

Captura de Pantalla 2022-09-21 a la(s) 6 50 41 p  m

You can prevent this behavior by adding this at the very beginning of your CSS file 🤓 
```
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```

File: index.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="socials">
+          <img src="./assets/github.png" alt="github" />
+          <img src="./assets/linkedin.png" alt="linkedin" />
+          <img src="./assets//Vector.png" alt="vector1" />
+          <img src="./assets//twitter.png" alt="twitter" />
+          <img src="./assets//Vector2.png " alt="vector2" />
```
Comment (GiuseppeTG) [inline index.html]: - [ ] When you hover over these images the cursor should become a pointer as they are supposed to be links to social media. You can set this in your CSS with `cursor: pointer` 👍

Comment (GiuseppeTG) [inline index.html]: - [ ] The list of icons should be a `ul` tag to group the images, and have a `li` tag per icon; each icon has a link (`a` tag). 
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.

**Review by GiuseppeTG — CHANGES_REQUESTED** [review]
Hi @vanelnw  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
✅ Professional PR
✅ Good commit messages
✅ Nice style

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GiuseppeTG** [comment]
- [ ] This is a general requirement. Please use the same colors and fonts as the Figma design. 👀 
Captura de Pantalla 2022-09-21 a la(s) 7 11 39 p  m
You can inspect the elements in Figma and see their exact values. 🤓

**Review by sumairq — APPROVED** [review]
### Hi @vanelnw , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**